site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md

```diff
@@ -0,0 +1,528 @@
+author: Guy Mast 
+id: security_analytics_with_wiz_and_snowflake
+summary: learn about the Wiz and Snowflake integration for analyzing Wiz data in Snowflake
+categories: cybersecurity,solution-examples,partner-integrations
+environments: web
+status: Hidden 
```

Comment:
@guy-mast-wiz Change the status to Published.

Reply:
Sure will do, how can I make this first hidden and not visible on https://quickstarts.snowflake.com/?

Reply:
What's the reason behind doing that? If it's not ready to be published, then let's wait to merge the PR.

Reply:
please ignore this, i have updated the status as you requested